pyproject.toml

```
@@ -33,6 +32,7 @@ dynamic = ['version']
 test = [
     'flake8',
     'pytest',
+    'pandas',
     'matplotlib',
 ]
 
```

```suggestion
full = [
    'ax-platform >= 0.3.4',
]
```

We could also add a `full` installation option for convenience, so that `pip install optimas[full]` would install all dependencies.